### DISCUSSION TITLE
Edit metadata (book title, author, etc)

### DISCUSSION THREAD

**Discussion by nicoleritzer** [PaulWoitaschek/Voice#1407] [Ideas]
Hi, as far as I can tell, there is no way to edit the displayed title and/or author of an audiobook in the app. Sometimes, the wrong info is displayed on the book selection/now playing screens, and I'd love the ability to change that.

**Comment by PaulWoitaschek** [comment]
It's true for the author, but you can change the name of the book though a long press on the book from the overview screen.

**Comment by sirsimon** [comment]
Yes, editing the author would be great!

**Comment by ahstro** [comment]
Would maybe make sense to add this to the existing [title edit dialog](https://github.com/PaulWoitaschek/Voice/assets/3090888/5608bb3e-8189-4dda-b8a5-e936eaf06a57)? And just rename the long press menu option from "Name" to "Edit"?
Cursory glance on the code (`edit_book_title` and `EditBookTitleDialog.kt`) suggest there might be some minor refactoring needed, and I'm at work and will forget this by the time I'm home, so can't promise I'll start digging 🤷
